Makefile

```diff
@@ -41,13 +41,16 @@ help:
 ##@ Dependencies
 
 .PHONY: build-dev-deps
-build-dev-deps: ## Install dependencies for builds
+build-dev-deps: ## Install dependencies for development
 	go get github.com/mattn/goveralls
 	go get golang.org/x/tools/cover
 	go get github.com/modocache/gover
-	go get github.com/go-bindata/go-bindata/go-bindata
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ${GOPATH}/bin v1.19.1
 
+.PHONY: install-build-deps
+install-build-deps: ## Install dependencies for builds
```

Comment:
Initially, I named this target `build-deps` however for some reason `make help` didn't list the target with that name, so I renamed it to this one.